src/main/java/org/jitsi/videobridge/SctpConnection.java

```diff
@@ -943,8 +943,16 @@ public void onConnOut(SctpSocket s, byte[] packet)
                             packet);
                 }
 
-                // Send through DTLS transport
-                transformer.sendApplicationData(packet, 0, packet.length);
+                // Send through DTLS transport.
+                threadPool.execute(new Runnable()
```

Comment:
I'm not absolutely comfortable with sending each "packet" in a separate Thread of "threadPool". You said the TCP socket would block for very roughly 30 seconds. DominantSpeakerIdentification runs at an interval of 300 milliseconds which in the worst case would result in 100 "threadPool" Threads. I'd rather we use a packet queue and send it in a single Thread (of "threadPool"). Anyway, I'll think about it a tiny bit more.